packages/assets/package.json

```diff
@@ -27,7 +27,7 @@
     "access": "public"
   },
   "dependencies": {
-    "@atb-as/theme": "^10.3.2",
+    "@atb-as/theme": "^10.3.3",
```

Comment:
Did you bump the versions manually? This is done automatically when publishing the new version with `yarn pub` after this has been merged into the main branch :)

Reply:
And here I thought I was being smart.  Ill revert the bumping stuff